deploy-vm.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+
+if ! [[ $(sudo which docker) && $(sudo docker --version) ]]; then   #[ -x "$(command -v docker)" ];
+    # Install requirements (docker)
+    echo "### Removing existing docker libs"
```

Comment:
Not a big fan of echo statements like `echo "### Hello world"`. Can you please change the statements to something like: `echo "Hello world"`